categories/communication/rules-to-better-communication.md

```diff
@@ -27,6 +27,8 @@ index:
   - corridor-conversations
   - tofu
   - attention-to-detail
+  - poc-vs-mvp
```

Comment:
Hey @bradystroud 

Agreed this is important, but I don't think it belongs to "Communication" category

Reply:
I agree, removed